src/test/resources/bulk/fms/03_phenotype_annotation/AF_01_all_fields_primary_allele_annotation.json

```diff
@@ -0,0 +1,40 @@
+[
```

Comment:
Sorry to be picky, but can we rename this file to AF_02_all_fields_primary_allele_annotation.json for consistency with the naming convention.  I also missed this on Adam's PR for the gene phenotype annotations, so would you mind renaming AF_01_all_fields_primary_gene_annotation.json to AF_03_all_fields_primary_gene_annotation.json.   The order of AGM/Allele/Gene doesn't matter, but having sequential numbers makes it easier to spot the correct test file at a glance rather than them all starting with AF_01.